nemo/export/tensorrt_llm.py

```diff
@@ -353,17 +366,38 @@ def export(
                     from megatron.core.export.trtllm.trtllm_helper import TRTLLMHelper
                     from tensorrt_llm.layers import MoeConfig
 
+                    use_embedding_sharing = model_configs.get("share_embeddings_and_output_weights", False)
```

Comment:
This overrides parameter `use_embedding_sharing` passed to `TensorRTLLM.export(...)`. Should it be removed from signature?

Reply:
Also: how about keeping the original name `share_embeddings_and_output_weights`?